tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
-envlist = py{27,35,36},docs,public_names
+envlist = py{27,35,36},lint,docs,public_names
```

Comment:
@zachmullen Since it's part of [the default `envlist`](https://tox.readthedocs.io/en/latest/config.html#conf-envlist), simply running `tox` will kick off the `tox -e lint` test too. However, I realize that CI always asks for specific Tox environments, so you are correct, I need to fix this. Oops. :roll_eyes: